README.md

```diff
@@ -43,3 +43,54 @@ To run local server in production mode:
 ```sh
 pnpm ssdeploy run
 ```
+
+
+## HTTP API
+You can use the site API in your own applications.
+
+```
+https://browsersl.ist/api/browsers?q=defaults&region=alt-ww
+```
+
+- `q` — query. Examples are available [on the website](https://browsersl.ist) or in the [browserslist repository](https://github.com/browserslist/browserslist#full-list). `defaults` by default.
+- `region` — region code. List of all region codes can be found at [caniuse-lite/data/regions](https://github.com/browserslist/caniuse-lite/tree/main/data/regions). `Global` by default.
```

Comment:
Yes, Global*

but it's better to specify alt-ww

Reply:
I wrote `alt-ww'. 

Otherwise it is misleading. Users will not understand why `Global` is not in the Сan I Use regions list. And why `Asia` doesn't work.